apps/hooks/pipeline/code-artifact-docker/index.ts

```diff
@@ -6,6 +6,7 @@ import { InvocationRequest } from 'aws-sdk/clients/lambda';
 interface CAHandlerEvent extends InvocationRequest {
   codeArtifactRepo: string;
   codeArtifactDomain: string;
+  codeArtifactNamespace: string;
```

Comment:
probably don't need this, since we read the namespace from `process.env`

Reply:
true

Reply:
srry im dumb, we need this